dclab/rtdc_dataset/writer.py

```diff
@@ -217,7 +217,15 @@ def store_feature(self, feat, data, shape=None):
             else:
                 raise ValueError(f"Bad shape for {feat}! Expeted {shape}, "
                                  + f"but got {data.shape[1:]}!")
-            self.write_ndarray(group=events, name=feat, data=data)
+                
+            # above code ensure that the shape of the (plugin or temp) data 
+            # must be 3 dimensions --> (None, dim1, dim2).
+            # If dim1 and dim2 are greater than 3, it would be considered as 
```

Comment:
If **the size of** dim1 and dim2 ...

You could also say (z, y, x) instead of dim2, dim2 as they are a bit comfusing.

Reply:
Instead of (x,y,z), I used rows and columns notation of ndarrays. Is it clear now?